src/reusable-blocks/class-reusable-blocks-integration.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace WPML\PB\Gutenberg;
+
+class Reusable_Blocks_Integration implements Integration{
+
+	const POST_TYPE = 'wp_block';
```

Comment:
@strategio This const is not used in this class so it probably belongs somewhere else. I would move to the `Reusable_Blocks_Translation` where it's used for now.